src/pages/detail/Detail.js

```diff
@@ -1,7 +1,374 @@
+import "slick-carousel/slick/slick-theme.css";
 
 const Detail = () => {
-  return <div>Detail</div>;
+  // const [isModal, setIsModal] = useState(false);
+  const [modalIsOpen, setIsOpen] = React.useState(false);
+  // const [closeModal, SetCloseModal] = useState();
+  const [imageIndex, setImageIndex] = useState(0);
+
+  const [bottomPage, setBottomPage] = useState("객실안내");
+  const [room, setRoom] = useState({});
+
+  useEffect(() => {
+    fetch("data/detailData.json", {
+      method: "GET",
+    })
+      .then(res => res.json())
+      .then(data => {
+        setRoom(data);
+        // console.log(data);
+      });
+  }, []);
+
+  // const modalClose = () => {
+  //   setIsModal(!isModal);
+  // };
+
+  // function openModal() {
+  //   setIsOpen(true);
+  // }
+
+  function afterOpenModal() {
+    // references are now sync'd and can be accessed.
+    // subtitle.style.color = "#f00";
+  }
+
+  function closeModal() {
+    setIsOpen(false);
+  }
+
+  let settings = {
+    dots: true,
+    infinite: true,
+    speed: 500,
+    slidesToShow: 3,
+    slidesToScroll: 3,
+    arrows: true,
+    autoplay: false,
+    autoplaySpeed: 5000,
+    centerMod: true,
+  };
+  return (
+    <Container>
+      <Maintop>
+        <AsideImages>
+          <AsideImage
+            src={room.image && room.image[imageIndex].image}
+            alt="roomImg"
+          />
+          <StyledSlider {...settings}>
+            {room.image?.map((list, idx) => (
+              <SliderImage
+                key={list.id}
+                src={list.image}
+                alt="room1"
+                onClick={() => setImageIndex(idx)}
+              />
+            ))}
+          </StyledSlider>
+        </AsideImages>
+        <AsideContent>
+          <AsideRoom>{room.name}</AsideRoom>
+          <MainText fontSize="20px" color="orange">
+            추천해요
+          </MainText>
+          <MainText>리뷰 700개</MainText>
+          <br />
+          <MainText>{room.add}</MainText>
+          <button onClick={() => setIsOpen(!modalIsOpen)}>
+            넷플릭스 프리존
+          </button>
+          <Modal
+            isOpen={modalIsOpen}
+            onAfterOpen={afterOpenModal}
+            onRequestClose={closeModal}
+            // style={customStyles}
+            contentLabel="Example Modal"
+            style={styleModal}
+          >
+            <ModalOne />
+            <ModalClose onClick={() => setIsOpen(false)}>X</ModalClose>
+          </Modal>
+          {/* {console.log(isModal)} */}
+          {/* {isModal && <Modal modalClose={modalClose} />} */}
+        </AsideContent>
+      </Maintop>
+      <UnderBottom>
+        <SideTab>
+          <RoomInfo onClick={() => setBottomPage("객실안내")}>
+            객실안내/예약
+          </RoomInfo>
+          <RoomInfo onClick={() => setBottomPage("숙소정보")}>
+            숙소정보
+          </RoomInfo>
+        </SideTab>
+        {/* {console.log(bottomPage)} */}
+        {bottomPage === "객실안내" && (
+          <div>
```

Comment:
해당 태그에 특별한 style이 없다면 Fragment로 작성해주시면 될 것 같습니다!
```suggestion
          <>
```

Reply:
네 수정완료했습니다!